content/concepts/asset-manager.md

```diff
@@ -1851,10 +1851,13 @@ The available media types that you can put in the `media_type` field are:
 :::
 
 ### The `boolean` attribute
+::: availability versions=SaaS editions=EE
+:::
+
```

Comment:
Part already marked as EE edition

Reply:
We would say that the attribute will be only available on SaaS and not in previous version <=7.0

Reply:
Do you know a different way to do it?